package.json

```diff
@@ -53,7 +51,7 @@
     "react": "16.14.0",
     "react-dom": "16.14.0",
     "react-helmet": "^6.1.0",
-    "react-intl": "^2.9.0",
```

Comment:
You update this package on 3 major versions. Did you make sure that the translation works as previously?

Reply:
After raising the translation version, I didn't notice any visible changes in the MFE Profile. I checked at your request